fe/src/project/AspNetCore/areas/Orders/components/StatusOrder/StatusOrder.js

```diff
@@ -22,58 +22,77 @@ function StatusOrder(props) {
-                                                    <TableCell>{item.deliveryFrom && <span>{moment(item.deliveryFrom).format("L")}</span>}</TableCell>
-                                                    <TableCell>{item.deliveryTo && <span>{moment(item.deliveryTo).format("L")}</span>}</TableCell>
-                                                    <TableCell>{item.moreInfo}</TableCell>
-                                                </TableRow>
-                                            )
-                                        })}
-                                    </TableBody>
-                                </Table>
-                            </TableContainer>
-                        </div>
-                    </section>
+                                                return (
+                                                    <TableRow key={index}>
+                                                        <TableCell><a href={item.productUrl}><img className="status-order__item-product-image" src={item.imageSrc} alt={item.imageAlt} /></a></TableCell>
+                                                        <TableCell>{item.sku}</TableCell>
+                                                        <TableCell>{item.name}</TableCell>
+                                                        <TableCell>{fabrics}</TableCell>
+                                                        <TableCell>{item.quantity}</TableCell>
+                                                        <TableCell>{item.externalReference}</TableCell>
+                                                        <TableCell>{item.deliveryFrom && <span>{moment(item.deliveryFrom).format("L")}</span>}</TableCell>
+                                                        <TableCell>{item.deliveryTo && <span>{moment(item.deliveryTo).format("L")}</span>}</TableCell>
+                                                        <TableCell>{item.moreInfo}</TableCell>
+                                                    </TableRow>
+                                                )
+                                            })}
+                                        </TableBody>
+                                    </Table>
+                                </TableContainer>
+                            </div>
+                        </section>
+                    </div>
                 </div>
-            </div>
+            }
+            {props.customOrderComment &&
+                <div className="mt-5">
+                    <h2 className="subtitle is-5 mb-2">{props.nonstandardOrderLabel}</h2>
+                    <div className="status-order__items">
+                        <TextField 
+                            value={props.customOrderComment}
+                            fullWidth={true}
+                            multiline={true}
+                            disabled={true}
+                            rows={3}
```

Comment:
**suggestion (non-blocking):** Move to constants to avoid magic strings

Reply:
I added to OrderFormConstants i think that will be okey, because this is meaning with order 😊